app/controllers/admin/collections_controller.rb

```diff
@@ -217,4 +217,40 @@ def destroy
       redirect_to admin_collection_path(@source_collection)
     end
   end
+
+  # POST /collections/1/batch_ingest
+  def batch_ingest
+    if @collection.dropbox.manifest_processable?
+      # Spawn off background job
+      AvalonJobs.collection_batch_ingest(@collection.id)
+      flash[:notice] = t('batch_ingest.initiated', name: @collection.name)
+    else
+      flash[:error] =
+        if @collection.dropbox.manifest_error?
```

Comment:
One other thing beyond the test suite that these are good for is introspection. If I run the 'methods' on this and get 'manifest_processable?', 'manifest_error?', etc. I get some insights into the states. If I get 'manifest_status', I don't get an indication of the states I can expect from this class.

Anyways, I just want to survive this PR and move on to other things, so I'll do it your way :smiling_imp: